2-Draft-Accepted/RFCxxxx-DSC-for-PowerShell7.md

```diff
@@ -0,0 +1,117 @@
+---
+RFC: RFCxxxx
+Author: Steve Lee
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: Desired State Configuration
+Comments Due: Nov 22, 2020
+Plan to implement: Yes
+---
+
+# DSC for PowerShell 7
+
+[Desired State Configuration](https://docs.microsoft.com/powershell/scripting/dsc/overview/overview) (DSC) is a platform in PowerShell
+enabling configuration as code.
+
+Currently, DSC (for Windows PowerShell) is tightly coupled with WMI on Windows and OMI on Linux.
+WMI is used to host the Local Configuration Manager (LCM) and WMI APIs are used to parse MOF which is used for both
+configuration files as well as schema files.
+
+In addition, it is not possible to write DSC resources that are cross platform compatible nor easily author DSC resources on Linux as
+they need to be in C++ or Python.
+Finally, third party solutions trying to leverage DSC resources need to still go through the LCM making it complicated to integrate with
+their agents or ecosystems.
+
+Unlike DSC for Windows PowerShell, DSC for PowerShell 7 is only a platform for building or integrating with configuration solutions.
+
+## Motivation
+
+    As a developer of configuration solutions,
+    I can easily leverage DSC resources,
+    so that my solution can manage Windows.
```

Comment:
"so that my solution can manage Windows." too restrictive IMO.
The real benefit as always to tap into the PowerShell ecosystem, community and consumers.
"so that my solution can leverage the PowerShell ecosystem and its users and contributors".